test/test_client.py

```diff
@@ -535,6 +535,16 @@ def test_client_options(self):
         self.assertIsInstance(c.options.retry_writes, bool)
         self.assertIsInstance(c.options.retry_reads, bool)
 
+    def test_validate_suggestion(self):
+        """Validate kwargs in constructor."""
+        self.assertRaises(ConfigurationError, MongoClient, auth="standard")
```

Comment:
We can replace this two asserts with one that uses `with self.assertRaisesRegex(...)`, for example: https://github.com/mongodb/mongo-python-driver/blob/6c88c732194335c8ec8e9c2c6fbd8c81d34151fc/test/test_client.py#L237-L238

Reply:
still todo.